api/provider/aws/README.md

```diff
@@ -242,12 +245,14 @@ func (e *EntityProvider) readResourceB(args) (*aws.ResourceB, error) {
 		return nil, err
 	}
 
+    // assuming `Describe()` returns `(ECSOutput, err)`
+    // and `ECSOutput.ResourceBs` is `[]*ResourceB`
 	output, err := e.AWS.ECS.Describe(input)
-	if err != nil {
-		return nil, err
-	}
+	if err != nil || len(output.ResourceBs) == 0 {
```

Comment:
This needs to be updated to reflect the changes we addressed earlier.

Reply:
✅